packages/client/src/pages/App/index.tsx

```diff
@@ -1,8 +1,24 @@
 import { Outlet } from 'react-router-dom'
 import './style.css'
 import { SnackbarProvider } from 'notistack'
+import { useAppDispatch } from '../../store/hooks'
+import { useEffect } from 'react'
+import { setUser } from '../../store/slices/userSlice'
+import { getUser } from '../../utils/scripts/api/yandexApi'
 
 function App() {
+  const dispatch = useAppDispatch()
+
+  useEffect(() => {
+    getUser().then(response => {
+      if (response?.data) {
+        dispatch(setUser(response.data))
+      } else {
+        dispatch(setUser(null))
+      }
+    })
```

Comment:
_Не критично к исправлению_

Тут можно сократить, сделав условный оператор прямо в `setUser`. Читаемость от этого не страдает, а даже улучшается

```suggestion
    getUser().then(response => {
      dispatch(setUser(response?.data || null))
    })
```